src/components/Breadcrumbs/Breadcrumbs.stories.mdx

```diff
@@ -39,34 +38,35 @@ import { getCategory } from "../../utils/componentCategories";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.0.3`    |
-| Latest            | `0.23.0`   |
+| Latest            | `0.25.1`   |
 
 <Description of={Breadcrumbs} />
 
 The `Breadcrumbs` component is a navigation element that provides a breadcrumb path that reflects the site structure and allows a user to navigate to any page available in the breadcrumb hierarchy.
 
-<Preview withToolbar>
+<Canvas withToolbar>
   <Story
-    name="Basic"
+    name="Breadcrumbs Props"
     args={{
-      breadcrumbs: [
-        { url: "#", text: "Parent" },
-        { url: "#", text: "Child" },
-        { url: "#", text: "Grandchild" },
+      breadcrumbsData: [
+        { url: "http://parent.com", text: "Parent" },
```

Comment:
A hash would be better because those are real links and clicking on them takes you to those real sites.

Reply:
Got it.